### PR TITLE
Ray integration docs

### PR DESCRIPTION
**Pull request by RamanaReddy8801**
**Ticket** : https://issues.newrelic.com/browse/NR-249391

**Description**: Ray - created new integration documentation for Ray integration.

**Preview image**:

screencapture-localhost-8000-docs-infrastructure-host-integrations-host-integrations-list-ray-integration-2024-03-20-12_26_37

### CODE REVIEW

**Comment by homelessbirds** [comment]
We created a ticket for this work for next sprint starting April 1st.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/ray-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+<Steps>
+    <Step>
+## Install the infrastructure agent
+
+To use the Ray integration, you need to first [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. The infrastructure agent monitors the host itself, while the Ray integration extends your monitoring with data specific to Ray clusters.        
+    </Step>
+    <Step>
+## Install the Prometheus integration
+1. Download the latest Prometheus release from the [Prometheus download page](https://prometheus.io/download/). Select the appropriate version for your operating system and architecture. For Linux, you'll likely choose the linux-amd64 version. Copy the download link for the tarball (`.tar.gz` file).
```
Comment (nbaenam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/ray-integration.mdx:33]: I'd add ``to linux-amd64, as follows `linux-amd64`.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/ray-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        config:
+        standalone: false
+        # Defaults to true. When standalone is set to `false`, `nri-prometheus` requires an infrastructure agent to send data.
+        emitters: infra-sdk
+        # When running with infrastructure agent emitters will have to include infra-sdk
+        cluster_name: Ray_Metrics
+        # Match the name of your cluster with the name seen in New Relic. 
+        targets:
+            - description: Ray_Metrics
+            urls: ["http://<YOUR_HOST_IP>:64747/metrics", "http://<YOUR_HOST_IP>:44217/metrics", "http://<YOUR_HOST_IP>:44227/metrics"]
```
Comment (nbaenam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/ray-integration.mdx:72]: I think we shouldn't use `<` or `>` if we follow our style guide.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/ray-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,153 @@
+---
+title: Ray integration
+tags:
+   - Ray integration
+   - New Relic integrations
+metaDescription: Install our Ray dashboards and see your ray data in New Relic.
```
Comment (nbaenam) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/ray-integration.mdx:6]: I'd write Ray instead of ray.

**Review by nbaenam — DISMISSED** [review]
I added some suggestions! Feel free to take them or leave them🙂

**Pull request merged**